lua/kommentary/init.lua

```diff
@@ -5,49 +5,98 @@ This module handles the initialization of the plugin.
 ]]
 local kommentary = require("kommentary.kommentary")
 local config = require("kommentary.config")
+local util = require("kommentary.util")
 local M = {}
+local context = util.enum({"line", "visual", "motion", "init"})
```

Comment:
```suggestion
local M = {}
local context = config.context
```
This is a very, very bad idea, since we're defining this *enum* twice per hand in two different files, we should really have one definition for it and then access that one from everywhere. I think this one makes a lot more sense in config.

This also renders the `util.lua` import unnecessary.